tests/examplefiles/haml/index.html.haml.txt

```diff
@@ -0,0 +1,17 @@
+!!! XML
```

Comment:
Files in `examplefiles/` usually end in the normal extension of the language, unlike `snippets/`.